torch/csrc/autograd/profiler.h

```diff
@@ -35,11 +35,21 @@ constexpr inline size_t ceilToMultiple(size_t a, size_t b) {
   return ((a + b - 1) / b) * b;
 }
 
+#if defined(__MACH__) && !defined(CLOCK_REALTIME)
+#include <sys/time.h>
+// clock_gettime is not implemented on older versions of OS X (< 10.12).
+// If implemented, CLOCK_REALTIME will have already been defined.
+#endif
+
 inline int64_t getTime() {
 #ifdef _WIN32
   using namespace std::chrono;
   using clock = std::conditional<high_resolution_clock::is_steady, high_resolution_clock, steady_clock>::type;
   return duration_cast<nanoseconds>(clock::now().time_since_epoch()).count();
+#elif defined(__MACH__) && !defined(CLOCK_REALTIME)
+  struct timeval now;
+  gettimeofday(&now, NULL);
```

Comment:
Is `gettimeofday` expected to be much faster than `chrono` as well? Did you profile it? Otherwise we could use the same code as for Windows (the case above).

Reply:
I did not profile it.

Maybe it is better to simply use `std::chrono`